examples/basic.zig

```diff
@@ -27,4 +27,5 @@ fn route(verse: *Verse) Router.Error!BuildFn {
 fn index(verse: *Verse) Router.Error!void {
     try verse.quickStart();
     try verse.sendRawSlice("hello world");
+    try verse.finish();
```

Comment:
I'm writing verse because I don't want to duplicate the original std.http implementation. The API can't requires clients to make a call to any kind of "finish"

Reply:
Done